<launch>

    <!-- LOAD THE PARAMETER LAUNCH FILE (WITH ITS ARGS) ABD LAUNCHES THE HORIZON PUBLISHER NODE (which, based on the config files, reads the correct trajectory and sends it to XBot) -->
   
    <arg name="show_plots" default="true"/>
    <arg name="save_fig" default="true"/>

    <param name="/horizon/horizon_plotter/show_plots" type="bool" value="$(arg show_plots) "/>
    <param name="/horizon/horizon_plotter/save_fig" type="bool" value="$(arg save_fig) "/>

    <!-- Launch the plotter node -->
    <node name="horizon_plotter" pkg="awesome_leg_pholus" type="horizon_plot.py" output="screen"/> 

</launch>